src/Microsoft.AspNetCore.Mvc.ViewFeatures/Internal/AntiforgeryValidationFailedResult.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Antiforgery;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.AspNetCore.Mvc.ViewFeatures.Internal
```

Comment:
You would want this in a non-internal namespace

Reply:
Good point. I originally put this in Mvc.ViewFeatures only because that was the only project that already had a reference to Antiforgery (since I needed the `AntiforgeryOptions`).

But with that requirement gone now (see below), I guess I can just put this result into `Mvc.Core` with all the other result types.